### PR TITLE
add gruvbox-material-gtk

### PR DESCRIPTION
**Pull request by oxcl**
## Description of changes

this pull request adds [gruvbox-material-gtk](https://github.com/TheGreatMcPain/gruvbox-material-gtk), a theme for GTK, Gnome, Cinnamon, XFCE, Unity, Plank and Icons.
## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by AndersonTorres** [comment]
Read the contributing guidelines.

**Comment by oxcl** [comment]
> Related to / duplicate of #188402 (although this PR has not been merged for a _long_ time)

i saw it but i think that's only for the icons. this includes the theme as well.

**Comment by oxcl** [comment]
> Read the contributing guidelines.

i skimmed through it. which part should i read exactly?

**Review by AndersonTorres — APPROVED** [review]

**Review by AndersonTorres — COMMENTED** [review]
The commit message does not conform with contributing guidelines.

File: pkgs/data/themes/gruvbox-material-gtk/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{lib, stdenv, fetchFromGitHub, gtk-engine-murrine }: 
+
+stdenv.mkDerivation rec {
+  name = "gruvbox-material-gtk";
```
Comment (AndersonTorres) [inline pkgs/data/themes/gruvbox-material-gtk/default.nix:4]: where is the version?

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

**Pull request closed without merging**